model/SISCourseV.js

```diff
@@ -21,6 +21,7 @@ const courseSchema = new Schema({
       level: { type: String },
       tags: [{ type: String }],
       preReq: { type: Array },
+      postReq: { type: Array, default: [] },
```

Comment:
well it's not just any array, it's an array of 

{
                  courseId: matchedCourse.id,
                  number: matchedCourse.number,
                  title: matchedCourse.title,
                  credits: matchedCourse.versions[0].credits,
                  preReqs: preReq.Expression,
                }

Reply:
Nice to have: make new MongoDB model with these fields ^ 

or at least have some way to specify the schema of the objects in the postReq array

Reply:
will do, we weren't sure what info we wanted in the post req array when we first wrote this, but we do now.